uniprot-config/src/main/resources/return-fields-config/uniparc-return-fields.json

```diff
@@ -72,6 +72,16 @@
     "id": "names_&_taxonomy/proteome",
     "includeInSwagger": true
   },
+  {
+    "parentId": "names_&_taxonomy",
+    "childNumber": 6,
+    "itemType": "SINGLE",
+    "label": "Common taxons",
+    "name": "common_taxons",
+    "paths": ["commonTaxons"],
+    "id": "names_&_taxonomy/common_taxons",
+    "includeInSwagger": true
+  },
   {
```

Comment:
Please run the tests in "uniprot-config" folder. We might be verifying  number of fields in test